src/components/MyDiagramContainer.tsx

```diff
@@ -6,8 +6,10 @@ import { nodeSelected, nodeDeselected, removeNode, removeLink } from '../actions
 import { DiagramModel, LinkModel, ModelChangeEvent, ModelChangeEventType } from 'react-gojs';
 import { Action } from 'typescript-fsa';
 
-interface MyDiagramContainerStateProps {
+interface MyDiagramContainerProps {
```

Comment:
you should keep the name MyDiagramContainerStateProps because it is the type result of the mapStateToProps function.
Don't forget that the component is connected to redux